docs/src/main/paradox/scala/http/sse-support.md

```diff
@@ -0,0 +1,57 @@
+<a id="sse-support-scala"></a>
```

Comment:
I have a WIP branch somewhere to remove the IDs since as you point out they are not needed at the "page level".